### ISSUE TITLE
IE 10 support

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-a11y#130]
I think we lost that along the way. Cf screenshot.

![virtualbox_ie10 - win7_08_06_2016_21_19_58](https://cloud.githubusercontent.com/assets/426686/15907779/cf1bbee6-2dbe-11e6-90eb-a8305e2a7513.png)

**Comment by zorkow** [comment]
Different results in different renderers:
* HTML-CSS: See screen shot. Error:
```
Unable to get property 'w' of undefined or null reference
auto-collapse.js, line1 character 5566
```
* SVG: 4 math processing errors. Menu only visible on first error. No obvious error.
* CommonHTML: Mostly works. There are three math processing errors in the Struik sample. O/w/ speech is generated correctly. Walking and collapsing works as expected.

**Comment by zorkow** [comment]
Problem for HTML-CSS seems to be in line
SRE.width = jax.root.data[0].HTMLspanElement().parentNode.bbox.w;

**Comment by zorkow** [comment]
HTML-CSS: It appears that for the display elements the parentNodes do not have a bbox.

**Comment by zorkow** [comment]
Correction: It can cope with null but not with undefined as final argument.
So ```node.insertBefore(new, old || null);``` actually solves the problem.

**Comment by dpvc** [comment]
@zorkow, that sounds like a good solution.  I had always done an if-then to switch between `insertBefore()` and `appandChild()`, because of the earlier IE implementations of `insertBefore()`, but this looks better.

**Comment by zorkow** [comment]
OK the CommonHTML works now on IE10. I'll start investigating the other problems now.

**Comment by zorkow** [comment]
HTML-CSS also works fine. So it must have been the insertBefore problem.

**Comment by pkra** [comment]
Yay! Now let's just fix #119  -- can I haz SVG :cat:

**Comment by zorkow** [comment]
SVG works now find on IE10. 
NVDA seems to read the label twice, otherwise it works.

**Comment by pkra** [comment]
And also may have fixed a bug I was seeing on Edge before. Sweet.
